src/inmanta/ast/__init__.py

```diff
@@ -523,6 +539,14 @@ def importantance(self):
     def attach_compile_info(self, compiler: "Compiler") -> None:
         self.root_ns = compiler.get_ns()
 
+    def export(self) -> export.Error:
+        location: Optional[Location] = self.get_location()
+        return export.Error(
+            type=str(builtins.type(self)),
```

Comment:
This doesn't produce the expted result

`"type": "<class 'inmanta.parser.ParserException'>",`

I would expect `'inmanta.parser.ParserException'`